### ISSUE TITLE
(Wrong) Icons are read by certain screen readers

### ISSUE THREAD

**Issue by yatil** [w3c/wcag-em-report-tool#203]
Apparently glyphicons (that are used in Bootstrap) are set to glyph positions where other icons are by default. The document icon in the menu reports „Red Heart“ in Voice Over on MacOS X 10.10  – And while I like W3C to show some love, it might be confusing for screen reader users. I propose to add `aria-hidden="true"` to every instance of those icons (as the icon is redundant and comes with “regular text”).

See this video for details: http://dl.dropbox.com/u/4266/wcag-em-icons.mp4

**Comment by rvantonisse** [comment]
Icons should be inserted as presentational images instead, e.g. svg's.

**Comment by nitedog** [comment]
@hidde please confirm but I think we are not using these icons anymore in [version 3](https://github.com/w3c/wai-wcag-em-report-tool)?

**Comment by hidde** [comment]
Yes, corect! We are hiding presentational icons with `aria-hidden` and have alternative text where appropriate.

**Comment by hidde** [comment]
We recently launched the [3.0.0 version of the WCAG-EM Report Tool](https://w3.org/WAI/eval/report-tool), in which this issue is fixed.
